tests/menu_items_registry.test.ts

```diff
@@ -173,13 +173,18 @@ describe("Menu Item actions", () => {
     expect(dispatch).toHaveBeenCalledWith("PASTE_FROM_OS_CLIPBOARD", {
       target: target("A1"),
       text,
+      pasteOption: "onlyValue",
     });
   });
 
-  test("Edit -> paste_special -> paste_special_format", () => {
```

Comment:
Why change this test ? Only "paste_special_value from OS clipboard" should have been changed right ?

Reply:
Since we've changed the case to be called for `onlyFormat`, this test case would expect paste_special_format action to dispatch "PASTE_FROM_OS_CLIPBOARD" instead of "PASTE". Hence the change :)

Reply:
We should have a test testing `paste_special_format` calling `PASTE_FROM_OS_CLIPBOARD` and another one calling `PASTE` then :slightly_smiling_face: